pylint/pyreverse/dot_printer.py

```diff
@@ -93,7 +93,8 @@ def emit_node(
             f', fontcolor="{properties.fontcolor}"' if properties.fontcolor else ""
         )
         self.emit(
-            f'"{name}" [color="{color}"{fontcolor_part}{label_part}, shape="{shape}", style="{style}"];'
+            f'"{name}" [color="{color}"{fontcolor_part}{label_part}, shape="{shape}", \
+                style="{style}"];'
```

Comment:
This misplaced backslash leads to test failures as it essentially adds multiple whitespaces.
```suggestion
            f'"{name}" [color="{color}"{fontcolor_part}{label_part}, shape="{shape}"'
            f'style="{style}"];'
```
The problem is that the backslash is essentially in the middle of the string. Something like this should work.

Reply:
Fixed